artifacthub-repo.yml

```diff
@@ -0,0 +1,4 @@
+repositoryID: #TODO
```

Comment:
Please consider setting value here or remove this file. I don't think it's needed right now.